miner/worker.go

```diff
@@ -884,6 +903,10 @@ func (w *worker) commitTransaction(tx *types.Transaction, coinbase common.Addres
 
 	// do not do CCC checks on follower nodes
 	if w.isRunning() {
+		defer func(t0 time.Time) {
```

Comment:
`defer` is triggered when return. is this the expected behavior?

Reply:
update: yes.

Reply:
I think it's intended, as the name suggests this timer measures the whole duration of `commitTransaction`.